deploy/cloud/README.md

```diff
@@ -1,40 +1,69 @@
 # Cloud Deployment
 
 **Motivation**
-Provide an easy and deterministic way to setup latest cloud environment so it can be monitored and used properly.
+Provide an easy and deterministic way to set up latest cloud environment, so it can be monitored and used properly.
+
+This guide deploys both an Elastic cloud environment, and an AWS EKS cluster. To only deploy specific resources, check out the examples section.
 
 **Prerequisite**
-* [terraform](https://www.terraform.io/)
+* [Terraform](https://developer.hashicorp.com/terraform/downloads)
+* the AWS CLI, [installed](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html) and [configured](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-configure.html)
+* [AWS IAM Authenticator](https://docs.aws.amazon.com/eks/latest/userguide/install-aws-iam-authenticator.html)
+* the [Kubernetes CLI](https://kubernetes.io/docs/tasks/tools/install-kubectl/), also known as `kubectl`
+
 
 **How To**
 Create environment
 1. Create an [API token](https://cloud.elastic.co/deployment-features/keys) from your cloud console account.
+
     1.1 use the token `export EC_API_KEY={TOKEN}`
+
 2. run `cd deploy/cloud`
 3. run `terraform init`
-4. run `terraform apply --auto-approve` to create the environment from the latest version (the latest version is vary in cloud/regions combinations).
+4. run `terraform apply --auto-approve` to create the environment from the latest version (the latest version is varying in cloud/regions combinations).
```

Comment:
nit: combine steps makes it easier to copy
```suggestion
2. to create the environment from the latest version (the latest version is varying in cloud/regions combinations).
   ```bash
   cd deploy/cloud
   terraform init
   terraform apply --auto-approve
   ```
```